templates/lobby.html

```diff
@@ -483,6 +484,8 @@
             }
             if (ready.wordHints && ready.wordHints.length) {
                 applyWordHints(ready.wordHints)
+            } else {
+                displayPlayerChoosingWord(parsed.data.players);
```

Comment:
This might cause showing the dialog at the end of the game, or am i mistaken?

Reply:
I just tested it, it doesn't seem to!